src/header_variables_r11.spec

```diff
@@ -323,3 +323,7 @@
     return 0;
   FIELD_CAST (VISRETAIN, RS, B, 70); //ok
 
+  /* TODO Unknown 18 bytes */
+  DEBUG_HERE //0x6bd-0x6ce
+  dat->byte = 0x6cf;
```

Comment:
I think, we should better store the unknown slack in some buffer. For encode

Reply:
Hm, this is probably not part of header.
I have two versions of DWG, which have 204 and 205 header variables. And difference is VISRETAIN value.

Reply:
I suspect it's a similar kind of sentinel, some magic, unique bytes after the headers, and before the entities